### PR TITLE
[baxtereus] fix bug in :command-grasp and :calib-grasp

### PR DESCRIPTION
**Pull request by knorth55**
- pass arm as symbol in `:commamd-grasp`

```lisp
(send *ri* :command-grasp "calibrate" :arms :pos 100)
```
- pass pos in `calib-grasp`
    - this is a typo

### CODE REVIEW

File: jsk_baxter_robot/baxtereus/baxter-interface.l [hunk 1]
```diff
@@ -183,20 +177,20 @@
...
-     (if pos
-         (send ee-cmd :args (format nil "{\"position\": ~A}" pos)))
-     (ros::publish (format nil "/robot/end_effector/~A_gripper/command" arm) ee-cmd)
-     (setq gripper-sequence-id (1+ gripper-sequence-id))
-     )
-   )
+   (dolist (tmp-arm (if (eq arm :arms) (list :larm :rarm) (list arm)))
+     (let ((ee-cmd (instance baxter_core_msgs::EndEffectorCommand :init)))
+       (if (eq tmp-arm :rarm)
+       (send ee-cmd :id right-gripper-type)
```
Comment (pazeshun) [inline jsk_baxter_robot/baxtereus/baxter-interface.l]: indent?

Comment (knorth55) [inline jsk_baxter_robot/baxtereus/baxter-interface.l]: fixed

**Comment by k-okada** [comment]
```
   (:command-grasp
     (cmd arm &key ((:pos pos) nil))
     (ros::spin-once)
 ```
BTW, what are you expecting to call `ros::spin-once` ?

**Comment by knorth55** [comment]
Hmm... I have no idea.  The line was added in this commit https://github.com/jsk-ros-pkg/jsk_robot/commit/45c821ad56184b776d04fd81c7d822c60c97958d.
I think it is not needed.

**Comment by k-okada** [comment]
please remove that line and see if we have any trouble

**Pull request merged**